src/nrniv/netpar.cpp

```diff
@@ -836,31 +841,19 @@ static void mk_localgid_rep() {
 	// perfect hash functions, or even simple Vectors.
 	localmaps_ = new Gid2PreSyn*[nrnmpi_numprocs];
 	localmaps_[nrnmpi_myid] = 0;
-	for (i = 0; i < nrnmpi_numprocs; ++i) if (i != nrnmpi_myid) {
-		// how many do we need?
-		sbuf = rbuf + i*(ngidmax + 1);
-		ngid = *(sbuf++); // at most
-		// of which we actually use...
-		for (k=0, j=0; k < ngid; ++k) {
-			if (gid2in_ && gid2in_->find(int(sbuf[k]), ps)) {
-				++j;
-			}
-		}
-		// oh well. there is probably a rational way to choose but...
-		localmaps_[i] = new Gid2PreSyn((j > 19) ? 19 : j+1);
```

Comment:
This is just a cleanup/simplification that's unrelated to the other changes, right?

Reply:
Right. The old hash table implementation encouraged some thought about how many slots would be appropriate for initialization. I guess ```std::unordered_map``` allows that, but I assume (or rather don't know) if it is worthwhile to consider it in this case.